src/components/cast/cast.js

```diff
@@ -58,17 +61,21 @@ class Cast extends Component {
   render(props: any): ?React$Element<any> {
     if (props.isCasting || props.isCastAvailable) {
       return (
-        <div
-          className={style.controlButtonContainer}
-          onClick={() => this.onClick()}
-          onKeyDown={e => {
-            if (e.keyCode === KeyMap.ENTER) {
-              this.props.updateBackdropVisibility(true);
-              this.props.player.startCasting().catch(() => this.props.updateBackdropVisibility(false));
-            }
-          }}>
-          <google-cast-launcher className={style.castButton} tabIndex="0" />
-        </div>
+        <Tooltip label={this.props.castText}>
+          <div
```

Comment:
why doesn't it button?

Reply:
This is old code - when working on the accessibility i tried changing it to button but the css was ruined so i rolled it back